h/templates/admin/groups.html.jinja2

```diff
@@ -16,6 +16,8 @@
           <th>URL</th>
           <th>Created by</th>
           <th>Members</th>
+          <th>Type</th>
```

Comment:
While I was in the template, I figured it wouldn't hurt to expose group type in the table.